analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -528,30 +530,81 @@ def postprocess(self, accumulator):
+                           "sip3d": e.Muon_sip3d,
+                           "pfRelIso04_all": e.Muon_pfRelIso04_all}.Zip()\
+                        .Where(lambda muon: (muon.pt > 30 
+                                             and abs(muon.eta) < 2.1 
+                                             and muon.tightId 
+                                             and muon.pfRelIso04_all < 0.15)).Count()== 1)\
+                        .Where(lambda f: {"pt": f.Jet_pt, 
+                                          "eta": f.Jet_eta,
+                                          "jetId": f.Jet_jetId}.Zip()\
+                               .Where(lambda jet: (jet.pt > 30 
+                                                   and abs(jet.eta) < 2.4 
+                                                   and jet.jetId == 6)).Count() >= 4)\
+                        .Where(lambda g: {"pt": g.Jet_pt, 
+                                          "eta": g.Jet_eta,
+                                          "btagCSVV2": g.Jet_btagCSVV2,
+                                          "jetId": g.Jet_jetId}.Zip()\
+                        .Where(lambda jet: (jet.btagCSVV2 >= 0.5 
+                                            and jet.pt > 30
+                                            and abs(jet.eta) < 2.4) 
+                                            and jet.jetId == 6).Count() >= 1)
+    selection = cuts.Select(lambda h: {"Electron_pt": h.Electron_pt,
+                                       "Electron_eta": h.Electron_eta,
+                                       "Electron_phi": h.Electron_phi,
+                                       "Electron_mass": h.Electron_mass,
+                                       "Electron_cutBased": h.Electron_cutBased,
+                                       "Electron_sip3d": h.Electron_sip3d,
+                                       "Muon_pt": h.Muon_pt,
+                                       "Muon_eta": h.Muon_eta,
+                                       "Muon_phi": h.Muon_phi,
+                                       "Muon_mass": h.Muon_mass,
+                                       "Muon_tightId": h.Muon_tightId,
+                                       "Muon_sip3d": h.Muon_sip3d,
+                                       "Muon_pfRelIso04_all": h.Muon_pfRelIso04_all,
+                                       "Jet_mass": h.Jet_mass,
+                                       "Jet_pt": h.Jet_pt,
+                                       "Jet_eta": h.Jet_eta,
+                                       "Jet_phi": h.Jet_phi,
+                                       "Jet_qgl": h.Jet_qgl,
+                                       "Jet_btagCSVV2": h.Jet_btagCSVV2,
+                                       "Jet_jetId": h.Jet_jetId,
+                                       "event": h.event,
+                                      })
+    if USE_INFERENCE:
+        return selection
+    
+    # some branches are only needed if USE_INFERENCE is turned on
```

Comment:
```suggestion
    # some branches are not needed if USE_INFERENCE is turned off
```

This is really minor, but as the next line describes the "off" case, I think this is clearer.